nimsuggest/tests/tisolated.nim

```diff
@@ -0,0 +1,11 @@
+# Regression test for server initialazed with project file but query another
+# file that not indexed with project's graph. this covers below use cases
+# 1. a new file under project directory structure not be imported nor included
+#    by compiled files.
+# 2. a file that used by explicitly importing in another project.
```

Comment:
I updated the phrasing, I'm pretty sure I got the meaning right.

```suggestion
# Regression test for querying a file/module that is not the project file/module
# or part of the transitive closure of all dependencies (via import or include).
# In such a case the queried file is not indexed within the project's graph.
#
# Covered use cases:
# 1. a new file under project directory structure not imported nor included by
#    compiled files.
# 2. a file that's used by explicitly importing in another project.
```

Regression test looks good.